src/data/nat/parity.lean

```diff
@@ -183,6 +183,9 @@ begin
     apply even_mul_succ_self }
 end
 
+lemma even_prime_eq_two {p : ℕ} (he : even p) (hp : prime p) : p = 2 :=
+((prime_dvd_prime_iff_eq prime_two hp).mp (even_iff_two_dvd.mp he)).symm
```

Comment:
Here's a golf, but perhaps there's not much point in keeping this lemma.
```suggestion
hp.even_iff.mp he
```

Reply:
I agree, I'm going to delete this lemma